auth/auth.go

```diff
@@ -170,22 +168,10 @@ func TokenEndpoint(endpoint environments.AzureADEndpoint, tenant string, version
 	return
 }
 
-func scopes(env environments.Environment, api Api) (s []string) {
-	switch api {
-	case MsGraph:
-		s = []string{fmt.Sprintf("%s/.default", env.MsGraph.Endpoint)}
-	case AadGraph:
-		s = []string{fmt.Sprintf("%s/.default", env.AadGraph.Endpoint)}
-	}
-	return
+func scopes(api environments.Api) (s []string) {
+	return []string{fmt.Sprintf("%s/.default", api.Endpoint)}
```

Comment:
As from https://github.com/hashicorp/go-azure-helpers/pull/83#discussion_r749173708 - we should make a public helper for this - this'll be common enough that you'd want a struct exposing the `Endpoint` and `Resource` for a Resource (and a helper for `DefaultScopeForEndpoint`)?

Reply:
Great idea, have added these as methods on `environments.Api`